mantidproject/mantid
# fix GetEiT0atSNS bug with pump pulses

**Description of work.**

GetEiT0atSNS is now failing because pump pulses show up in the monitor data. This fix uses MaskBins to mask out bins around the pump pulses.

**To test:**

Run GetEiT0atSNS on SEQ run 181259. Ei should be around 18meV

Fixes #26281  

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
